mods/ra/rules/defaults.yaml

```diff
@@ -277,6 +277,8 @@
 		StandSequences: stand,stand2
 	DetectCloaked:
 		CloakTypes: Hijacker
+	AutoTarget:
+		ScanRadius: -1
```

Comment:
You don't even need this, since this is the default value. The default.yaml changes should be reverted.